prefix.go

```diff
@@ -182,13 +182,24 @@ func (i *ipamer) AcquireChildPrefix(parentCidr string, length uint8) (*Prefix, e
 	var prefix *Prefix
 	return prefix, retryOnOptimisticLock(func() error {
 		var err error
-		prefix, err = i.acquireChildPrefixInternal(parentCidr, length)
+		prefix, err = i.acquireChildPrefixInternal(parentCidr, "", length)
+		return err
+	})
+}
+
+func (i *ipamer) AcquireSpecificChildPrefix(parentCidr, childCidr string) (*Prefix, error) {
+	var prefix *Prefix
+	return prefix, retryOnOptimisticLock(func() error {
+		var err error
+		prefix, err = i.acquireChildPrefixInternal(parentCidr, childCidr, 0)
 		return err
 	})
 }
 
 // acquireChildPrefixInternal will return a Prefix with a smaller length from the given Prefix.
```

Comment:
Please add comment about childcidr

Reply:
What should I write? A simple explanation about what childcidr is?
And where should I put it? Line 194 or above the exported func?